src/index.jsx

```diff
@@ -25,6 +23,20 @@ subscribe(APP_INIT_ERROR, (error) => {
 });
 
 initialize({
+  handlers: {
+    config: () => {
+      mergeConfig({
+        USE_API_CACHE: process.env.USE_API_CACHE || null,
```

Comment:
interesting:  the || null pattern, as opposed to just dropping the OR clause here ? what would we expect? undefined I guess? would that work too or? I like `null` as a more explicit value as opposed accidental undefined, but I don't know what the convention we have been using is (there are camps of undefined vs nulls hence I ask) but wanted to check on conventions in the least

Reply:
`mergeConfig` had a code comment or docs somewhere that mentioned throwing an error if the values provided to `mergeConfig` are undefined, so I kept with convention and included the `|| null`.

Reply:
hmm ok pretty much a requirement then! then it's good to know. null is better than undefined in this scenario anyway